lib/repl.js

```diff
@@ -158,6 +160,7 @@ function REPLServer(prompt,
   self.last = undefined;
   self.breakEvalOnSigint = !!breakEvalOnSigint;
   self.editorMode = false;
+  self[kContextId] = undefined;
```

Comment:
Would be helpful to clarify that this ID refers to the context ID in the inspector protocol,

Reply:
done